frontend/src/components/common/LinkList.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import './Label.scss';
+import { Link } from 'react-router-dom';
+import { FaRegTimesCircle } from 'react-icons/fa';
+import { Search } from 'history';
+import { ListGroup } from 'react-bootstrap';
+
+export interface ILinkListItem {
+  search: Search;
+  pathName: string;
+  label: string;
+  key: string | number;
+  onRemoveItemClick?: () => void;
+}
+
+interface ILinkedListProps {
+  listItems: ILinkListItem[];
+  disabled?: boolean;
+  noItemsMessage: string;
+}
+
+/**
+ * Generic component that displays a list of links that open in a new tab. Requires input to be formatted into a list of {@link ILinkListItem}.
+ * @param {ILinkedListProps} param0
+ */
+export const LinkList: React.FunctionComponent<ILinkedListProps &
```

Comment:
We are starting to use this list of links in a lot of places, so created a shared component for it.